wallet/udb/vsp.go

```diff
@@ -13,8 +13,21 @@ var (
 	vspBucketKey = []byte("vsp")
 )
 
+const (
+	// VSP_FEE_PROCESS_STARTED represents the state which process has being
+	// called but fee still not paid.
+	VSP_FEE_PROCESS_STARTED   = 0
```

Comment:
SCREAMING_CAPS is not go style, UseNamesLikeThis.

These constants should also use a defined type, and the constants should be declared using iota.